pqtgo/generator.go

```diff
@@ -310,6 +310,14 @@ type %sIterator struct {
 	rows *sql.Rows
 	cols []string
 }
+type I%sIterator interface {
```

Comment:
Maybe it's a good idea to generate for each entity `<entity-name>Iterator` interface and struct that will implement this interface rename to `<entity-name>Rows`? We could avoid having naming workarounds and new struct name will correspond to [sql.Rows](https://golang.org/pkg/database/sql/#Rows)